test/e2e/helm_test.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package e2e
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/kubernetes/test/e2e/framework"
+	"sigs.k8s.io/e2e-framework/third_party/helm"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	admissionapi "k8s.io/pod-security-admission/api"
+)
+
+// Helm based test suite
+var _ = NFDDescribe(Label("helm"), func() {
+	f := framework.NewDefaultFramework("node-feature-discovery")
+	// To avoid the error of violating the PodSecurity
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
+
+	manager := helm.New(framework.TestContext.KubeConfig)
+	release_name := "node-feature-discovery"
+
+	Context("when deploying by helm", Ordered, func() {
+		JustBeforeEach(func(ctx context.Context) {
+			// Install local nfd chart
+			workingDir, err := os.Getwd()
+			chartPath := filepath.Join(workingDir, "../../", "deployment", "helm", "node-feature-discovery")
+			Expect(err).NotTo(HaveOccurred())
+			By("helm install " + release_name + " " + chartPath + " --namespace " + f.Namespace.Name + " --wait")
```

Comment:
https://onsi.github.io/ginkgo/#documenting-complex-specs-by

Let's re write this "By"

Reply:
done